test/go2_test.cpp

```diff
@@ -162,5 +177,27 @@ TEST(SerowTests, Go2Test) {
                   << std::endl;
         std::cout << "CoM external forces " << state->getCoMExternalForces().transpose()
                   << std::endl;
+
+        // Store the results into an eigen matrix to be saved
+        if (STORE_PREDICTIONS) 
+        {
+            if (timestamp - prev_timestamp < 0){
+                sec_timer += 1.;
```

Comment:
+1?

Reply:
Yes, if time has moved backwards 0.998, 0.999 , 0.001 then add one second to the sec_timer else you are in the same second duration (e.g. third second 3 + 0.580, 3 + 0.581, 3 + 0.582 .... 3 + 0.999, 4 + 0.001). This is for plotting in continuous one-directional timeline